### PR TITLE
static pages

### PR DESCRIPTION
**Pull request by trunghieu0211**

### CODE REVIEW

**Comment by trunghieu0211** [comment]
[o] eslint: success
[o] reek: success
[o] bundle-audit: success
[o] rubocop: success
[o] scss-lint: success
[o] brakeman: success
[o] rails_best_practices: success
[o] rspec: success

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import 'bootstrap-sprockets';
```
Comment (SonNguyen1604) [inline app/assets/stylesheets/custom.scss]: sắp xếp

**Comment by trunghieu0211** [comment]
ready

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (thanhhungchu95) [inline app/assets/stylesheets/custom.scss]: Áp dụng với rule thôi mà anh (yaoming)

**Pull request merged**